Source/WebCore/rendering/style/StyleGridData.h

```diff
@@ -97,8 +97,8 @@ class StyleGridData : public RefCounted<StyleGridData> {
     void setRows(const GridTrackList&);
     void setColumns(const GridTrackList&);
 
-    const Vector<GridTrackSize>& gridColumns() const { return m_gridColumns; }
-    const Vector<GridTrackSize>& gridRows() const { return m_gridRows; }
+    const Vector<GridTrackSize>& gridColumns() const { return m_gridColumnTrackSizes; }
```

Comment:
It feels like we should rename these, and `RenderStyle::initialGridColumns()` too?